src/main/java/org/disq_bio/disq/HtsjdkVariantsRddStorage.java

```diff
@@ -155,37 +178,12 @@ public void write(HtsjdkVariantsRdd htsjdkVariantsRdd, String path, WriteOption.
       tempPartsDirectory = path + ".parts";
     }
 
-    getSink(formatWriteOption, fileCardinalityWriteOption)
+    WriteOption.getSink(formatWriteOption, fileCardinalityWriteOption)
```

Comment:
In the case that the refactoring is done, this line will read: `fileCardinalityWriteOption.getSink(formatWriteOption)`, which is cleaner from my point of view. And it could be understood as: creates a sink with a concrete file cardinality using the format requested.